share/spice/google_plus/google_plus.js

```diff
@@ -1,33 +1,41 @@
-function ddg_spice_google_plus (api_result) {
+(function (env) {
     "use strict";
+    env.ddg_spice_google_plus = function(api_result){
 
-    if(!api_result || !api_result.items || api_result.items.length === 0) {
-        return Spice.failed("googleplus");
-    }
+        if(!api_result || !api_result.items || api_result.items.length === 0) {
+            return Spice.failed("google_plus");
+        }
 
-    Spice.add({
-        id: 'googleplus',
-        name: 'Social',
-        data: api_result.items,
-        meta: {
-            sourceName : 'Google+',
-            sourceUrl : 'http://plus.google.com',
-            itemType: "Google+ Profile" + (api_result.items.length > 1 ? 's' : '')
-        },
-        templates: {
-            group: 'products_simple',            
-            item_detail: false,
-            detail: false,
-            variants: {
-                tile: 'narrow'
+        Spice.add({
+            id: 'google_plus',
+            name: 'Social',
+            data: api_result.items,
+            meta: {
+                sourceName : 'Google+',
+                sourceUrl : 'http://plus.google.com',
```

Comment:
This should really link to a search for the query. Can fix in another PR or I can make an issue.